tar/bsdtar.c

```diff
@@ -1251,7 +1252,7 @@ configfile(struct bsdtar *bsdtar, const char *fname)
 		 * than failing to report an error if there really is a
 		 * configuration file.
 		 */
-		bsdtar_errc(bsdtar, 1, errno, "stat(%s)", fname);
+		bsdtar_errc(bsdtar, 1, errno, "configfile stat(%s)", fname);
```

Comment:
I added the "configfile" because I found the error message unclear (even when I was expecting to see it because I was testing that case).  Non-programmers (or more specifically, non-posix C programmers) don't know what "stat" means.